spotify_player/src/config/keymap.rs

```diff
@@ -32,299 +32,322 @@ pub struct ActionMap {
 
 impl Default for KeymapConfig {
     fn default() -> Self {
+        // Generate keymaps for `j` and `k` keys with a count in front
+        // i.e. 4 j -> SelectNextOrScrollDown { count: 4 }
+        fn generate_keymap_for_up_and_down() -> Vec<Keymap> {
+            let mut keymaps = vec![];
+
+            for i in 1..=9 {
+                keymaps.push(Keymap {
+                    key_sequence: format!("{i} j").as_str().into(),
+                    command: Command::SelectNextOrScrollDown { count: i },
+                });
+                keymaps.push(Keymap {
+                    key_sequence: format!("{i} k").as_str().into(),
+                    command: Command::SelectPreviousOrScrollUp { count: i },
+                });
+            }
+
+            keymaps
+        }
```

Comment:
this doesn't have to be a function, you can add the logic as-is after defining `keymaps` variable